share/jupyterhub/templates/login.html

```diff
@@ -20,7 +20,7 @@
   </a>
 </div>
 {% else %}
-<form action="{{login_url}}?next={{next}}" method="post" role="form">
+<form action="{{authenticator_login_url}}?next={{next}}" method="post" role="form">
```

Comment:
`authenticator_login_url` includes `?next=`, so this is getting appended twice. You can remove the `?next=` here.

Reply:
I missed it, thanks |